src/shared/libs/versions.json

```diff
@@ -1,6 +1,6 @@
 {
     "desktopBlacklist": ["0.3.4", "0.3.5", "0.3.6", "0.4.0", "0.4.1"],
     "mobileBlacklist": ["20", "21", "22", "25", "26", "27"],
-    "latestMobile": "29",
+    "latestMobile": "41",
```

Comment:
Wouldn't this trigger an update notification for beta users who can't access this version?

Reply:
It's not being merged to develop, but good point. This shouldn't be updated until after release